src/IndirectArrays.jl

```diff
@@ -46,4 +55,38 @@ end
     ret
 end
 
+@inline function Base.setindex!(A::IndirectArray, x, i::Int)
+    @boundscheck checkbounds(A.index, i)
+    idx = findfirst(A.values, x)
+    if idx == 0 || idx == nothing # findfird changed in 0.7
```

Comment:
"findfirst". Could also use `Compat.findfirst` now.

Reply:
Indeed. I've updated the PR.